docker/Makefile

```diff
@@ -0,0 +1,86 @@
+# Copyright 2023 The KubeStellar Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Global arguments
+IMG := kubestellar
+NAME := kubestellar
+PLATFORMS := linux/amd64,linux/arm64,linux/ppc64le
```

Comment:
What about our 390?

Reply:
@MikeSpreitzer I'll add the linux/s390x in the defaults

Reply:
@MikeSpreitzer sorry, I just realized that there is not a kcp tarball for s390x so I cannot create an image for it